plugins/roll.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: roll
+spec:
+  homepage: https://github.com/LifeWay/kubectl-roll-plugin
+  shortDescription: Rolling restart of all persistent pods in a namespace
+  version: v1.0.4
+  description: |
+    A utility to slowly delete all pods in a namespace that are
+    controlled by Deployments, StatefulSets, and DaemonSets. You must
+    correctly configure readiness probes to avoid any downtime.
```

Comment:
I'm not sure if this is implicit or obvious, but the rollout strategy also plays a role in avoiding downtime. If it's set to `recreate`, there will be downtime, only `RollingUpdate` is safe (https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy).